src/main/java/org/spongepowered/api/event/entity/EntityDeathEvent.java

```diff
@@ -29,10 +29,32 @@
 import org.spongepowered.api.event.cause.CauseTracked;
 import org.spongepowered.api.event.inventory.ItemDropEvent;
 import org.spongepowered.api.util.event.Cancellable;
+import org.spongepowered.api.world.Location;
 
 /**
  * Called when an {@link Entity} is killed or removed due to unload.
  */
 public interface EntityDeathEvent extends EntityEvent, CauseTracked, Cancellable, ItemDropEvent {
-    
+
+    /**
+     * Gets the location of the player's death.
+     *
+     * @return The {@link Location} of the player's death
+     */
+    Location getLocation();
+
+    /**
+     * Gets the amount of experience that will be dropped on death.
+     *
+     * @return The amount of experience that will be dropped on death
+     */
+    int getDroppedExperience();
```

Comment:
There should be a note as to how this relates to the experience level and 'extra' amount.